MIP113/MIP113.md

```diff
@@ -186,28 +187,46 @@ Content: CVC Creation
 
     ¤¤¤
 
-##### 4.5: CVC Decisions
+##### 4.7: CVC Decisions
 
 CVC Decisions are formal actions that can be taken by CVCs.
-* **4.5.1:** CVC Decisions are proposed through a signed message to the Maker Governance Forum, from a CVC Member.
----
-* **4.5.2:** CVC Decisions are voted on during a one week period. CVC Members vote by posting a signed message referring the proposal and containing either "For" or "Against". Voting weight is based on the verified MKR of the CVC Member.
+
+* **4.7.1**: CVC Decisions are made via a signed message posted publicly to the Maker Governance Forum.
+	* **4.7.1.1**: Any CVC member may post this signed message.
+	* **4.7.1.2**: The signed message must contain a clear record of the decision particulars.
```

Comment:
Worth getting more specific here? The constitution states the votes need to be according to recognized MKR weight, should we include that in the instructions to save time later?

Reply:
It's specifically 'simple majority' in the scope. Its a case where the constitution and scope differ. I considered changing it, but in the interests of keeping the scope changes minimal, I kept that the same.